### PR TITLE
Updated readme.

### PR DESCRIPTION
**Pull request by julesfowler**
The data download is getting rather tiresome, so I figured it needed to be well documented in our README.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -137,6 +137,41 @@ or, clone the repository directly from GitHub:
   cd exoctk
   python setup.py [install|devlop]
 
+Obtain the ``exoctk`` Data
```
Comment (bourque) [inline README.rst]: I notice that this title is slightly larger than the one for `Obtain the ``exoctk`` Package`.  Is that intentional?

**Comment by julesfowler** [comment]
@bourque or @hover2pi I think this looks good. Also literally all I changed was text of the readme, so I'm tempted to say we should not care to wait about the travis builds. Could one of you review?

File: README.rst [hunk 2]
```diff
@@ -137,6 +137,42 @@ or, clone the repository directly from GitHub:
...
+  similar depending on version.) 
+- For Windows, use the default application to unzip the file (Extract or
+  similar depending on version.)
+- For Linux, use the java unzip :
+
+::
+
+    jar -xf exoctk_data.zip
+
+- If you try to use ``unzip``, ``7zip``, or ``gzip``, the files will likely be corrupted.
```
Comment (bourque) [inline README.rst]: For some odd reason, I see this line as split up onto two lines when viewing the file.

File: README.rst (hunk 1 above)
Comment (julesfowler) [reply README.rst]: Nice catch. That is not intentional, let me take a look.

File: README.rst (hunk 2 above)
Comment (julesfowler) [reply README.rst]: Screen Shot 2019-08-06 at 5 00 23 PM

This line appears as broken for you?

File: README.rst (hunk 1 above)
Comment (julesfowler) [reply README.rst]: @bourque Turns out my terminal text is too smoll to see the difference between `~` and `-`. Should look better now.

File: README.rst (hunk 2 above)
Comment (bourque) [reply README.rst]: Nevermind, it seemed to have fixed itself!

**Pull request merged**